src/components/Proposal/ProposalSummary/ProposalSummaryKnownGenericScheme.tsx

```diff
@@ -22,47 +23,143 @@ export default class ProposalSummary extends React.Component<IProps> {
     super(props);
   }
 
-  public render() {
+  public render(): any {
     const { proposal, detailView, transactionModal, genericSchemeInfo } = this.props;
-    if (genericSchemeInfo.specs.name === "DutchX") {
-      return <ProposalSummaryDutchX {...this.props} />;
-    }
+
+    return this.renderFormSummary(genericSchemeInfo, proposal, detailView, transactionModal);
+    
+    // const proposalSummaryClass = classNames({
```

Comment:
@jellegerbrandy I don't understand what this code is for, whether it is ever executed, so I commented it out.  Suggestions?

Reply:
Yeah, but I believe that fact will have been detected before ever reaching this component.

Reply:
I am not following you.  When the proposal regards a function call that is not recognized, we should tell the user that:  "Unknown function call, it looks like this: ..".  That is what this commeted code is doing.